### ISSUE TITLE
Will this support MacOS  apps

### ISSUE THREAD

**Issue by ankushom19** [helloSystem/hello#111]
Will this support MacOS  apps

**Comment by probonopd** [comment]
* Many applications are written using cross-platform toolkits like Qt. Those usually have Mac, Windows, and Linux versions and most of them can run on FreeBSD (with recompilation), and hence on helloSystem
* Theoretically, helloSystem could run macOS apps originally written in ObjectiveC for AppKit that are ported to and compiled with [GNUstep](https://en.wikipedia.org/wiki/GNUstep) if the GNUstep libraries are installed - unfortunately that are not many macOS apps that are ported to and compiled with it in real life
* As for running existing binaries of macOS applications (without porting and recompiling) - the answer is no at this time. This might change only once something like [Darling](https://www.darlinghq.org/) progresses significantly and is running on FreeBSD

So, in summary, this is not something the helloSystem project is working on, but if technology would emerge that would one let run macOS applications on FreeBSD (similar to how WINE lets one run Windows appiications on other platforms) we would certainly look into integrating it.
